data/tribes/wares/armor/init.lua

```diff
@@ -39,6 +39,10 @@ dirname = path.dirname(__file__)
 --
 --    **default_target_quantity**: A table listing the default target quantity
 --    for each tribe's economy. For example, ``{ atlanteans = 3, empire = 1 }``
+--    This parameter is optional. If not set the economy will always demand this ware.
```

Comment:
That's not correct. The table always has to be specified, but it may be empty.

Reply:
oops you are right